tox.ini

```diff
@@ -10,5 +10,5 @@ setenv =
 deps =
     .[all]
 commands =
-    py.test --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
+    py.test {posargs} --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
```

Comment:
I can't find that in `tox`'s docs and IMO `{posargs}` has incalculably superior readability and google-ability to `[]`. How about using `posargs` inserted at the point where `[]` is on the same +readability principle as "when writing a shell script for future use, use long argument flags rather than short ones"?